catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -161,18 +162,14 @@
     name = "Batch size",
     description =
         "Number of Metacards to ingest at a time. Change this argument based on system memory and Catalog Provider limits. [DEPRECATED: use --batchsize option instead]",
-    index = 1,
-    multiValued = false,
-    required = false
+    index = 1
```

Comment:
IntelliJ suggested "removing unused parameters". It did the refactoring for me.

Reply:
sometimes that makes it clearer though